test/views/incident/transportMeans/IdentificationNumberViewSpec.scala

```diff
@@ -47,6 +47,8 @@ class IdentificationNumberViewSpec extends InputTextViewBehaviours[String] {
 
   behave like pageWithHeading(identificationType.arg)
```

Comment:
Looks like we're passing `identification.arg` in where it's now not needed. Same for the title as well?